src/measure_theory/bochner_integration.lean

```diff
@@ -1243,7 +1243,7 @@ end
 
 variable {ν : measure α}
 
-lemma integral_add_meas {f : α → E} (hfm : measurable f) (hfi : integrable f (μ + ν)) :
+@[simp] lemma integral_add_meas {f : α → E} (hfm : measurable f) (hfi : integrable f (μ + ν)) :
```

Comment:
Is this a good simp lemma with the two arguments? Do we expect those to be provable by `simp` themselves?

Reply:
Probably not, removing `simp`.
```suggestion
lemma integral_add_meas {f : α → E} (hfm : measurable f) (hfi : integrable f (μ + ν)) :
```